src/screens/SignUp/styles.module.scss

```
@@ -0,0 +1,84 @@
+@import 'scss/variables/_colors';
+
+@import 'scss/layout';
+
+@import 'scss/components';
+
+@import 'scss/margins';
+
+.form-container {
+  background-color: $white-smoke;
+  border-top: 10px solid $deep-sky-blue;
+  height: 634px;
```

Try to use a base for your measurements I suggest 5px so try that width, heigh, margins and padding there are a multiple of this base 

```suggestion
  height: 635px;
```